libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnDialogEvent.schema

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
+    "$role": "unionType(Microsoft.ITriggerCondition)",
+    "title": "OnDialogEvent",
+    "description": "Defines a actions which are executed when a dialog event matching event is is true",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "./Microsoft.OnCondition.schema#"
+        },
+        {
+            "properties": {
+                "event": {
+                    "type": "string",
```

Comment:
Should onDialogEvent's event is a enum and onCustomizedEvent's event is a free string. 

I thought onCustomEvent is for listening for customized event
and onDialogEvent is for listening for system event.

Reply:
@tomlm was expecting to see something like this in the OnDialogEvent schema --
```json
"events": {
                    "type": "array",
                    "description": "Events to trigger this rule for",
                    "items": {
                        "type": "string",
                        "enum": [
                            "beginDialog",
                            "consultDialog",
                            "cancelDialog",
                            "activityReceived",
                            "recognizedIntent",
                            "unknownIntent",
                            "actionsStarted",
                            "actionsSaved",
                            "actionsEnded",
                            "actionsResumed"
                        ]
                    }
                }
```